nodewatcher/modules/frontend/ip_space/frontend.py

```diff
@@ -0,0 +1,34 @@
+from django.core import urlresolvers
+
+from nodewatcher.core import models as core_models
+from nodewatcher.core.frontend import components
+
+from . import views
+
+
+class IpSpaceCompontent(components.FrontendComponent):
+    @classmethod
+    def get_main_url(cls):
+        return {
+            'regex': r'^network/ip_space/$',
+            'view': views.IpSpace.as_view(),
+            'name': 'IpSpace',
+        }
+
+components.pool.register(IpSpaceCompontent)
+
+
+components.menus.get_menu('main_menu').add(components.MenuEntry(
+    label=components.ugettext_lazy("IP space"),
+    url=urlresolvers.reverse_lazy('IpSpaceCompontent:IpSpace'),
+))
+
+
+components.partials.register(components.Partial('ip_space_partial'))
+
+
+components.partials.get_partial('ip_space_partial').add(components.PartialEntry(
+    name='general',
+    template='ip_space.html',
```

Comment:
Is this ok that the template is at the top level? I think it should be better to move it under `network/ip_space/ip_space.html` or similar.